tests/test_downsample.py

```diff
@@ -0,0 +1,104 @@
+import unittest
+from unittest import TestCase
+
+import numpy as np
+
+from aspire.source import Simulation
+from aspire.utils import utest_tolerance
+from aspire.utils.matrix import anorm
+from aspire.utils.misc import gaussian_3d
+from aspire.volume import Volume
+
+
+class DownsampleTestCase(TestCase):
+    def setUp(self):
+        self.n = 128
+        self.dtype = np.float32
+
+    def tearDown(self):
+        pass
+
+    def testDownsample2D_EvenEven(self):
```

Comment:
Lots of repetition here. Can we either create another `_testDownsample(source, target)` or use `parameterized` in some way?